integrations_example.py

```diff
@@ -8,43 +8,54 @@
 # Version:      1.0
 #
 
-import json
 import argparse
 import csv
 import demisto
 
+
 def p(what):
     if verbose:
         print(what)
 
+
 def options_handler():
     parser = argparse.ArgumentParser(description='Integrations and commands')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
-    parser.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True, help="no extra prints")
-    parser.add_argument('-o', '--output', help='The output CSV file', default='integrations.csv')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
+    parser.add_argument('-q', '--quiet', action='store_false',
+                        dest='verbose', default=True, help="no extra prints")
+    parser.add_argument(
+        '-o', '--output', help='The output CSV file', default='integrations.csv')
     options = parser.parse_args()
     global verbose
     verbose = options.verbose
     return options
 
+
 def main():
     options = options_handler()
     c = demisto.DemistoClient(options.key, options.server)
     integrationsResponse = c.req('GET', 'settings/integration-commands', None)
     if integrationsResponse.status_code != 200:
-        raise RuntimeError('Error getting integration data - %d (%s)' % (integrationsResponse.status_code, integrationsResponse.reason))
-    integrations = sorted(integrationsResponse.json(), key=lambda m: m['category'].lower() + '---' + m['name'].lower())
+        raise RuntimeError('Error getting integration data - %d (%s)' %
+                           (integrationsResponse.status_code, integrationsResponse.reason))
+    integrations = sorted(integrationsResponse.json(
+    ), key=lambda m: m['category'].lower() + '---' + m['name'].lower())
```

Comment:
This is another odd place to add a newline in the middle of a function call?

Reply:
fix strange indent (49e8339).